tests/tools/test_dist_part.py

```diff
@@ -222,6 +222,13 @@ def test_chunk_graph_vector_rows(num_chunks, vector_rows):
     )
 
 
+"""
```

Comment:
If this functionality is no longer supported, do we still to keep these code or just remove them at all as a clean-up?

Reply:
Done.